model/src/test/kotlin/com/testerum/model/resources/http/response/XmlBodyHttpResponseTest.kt

```diff
@@ -0,0 +1,188 @@
+package com.testerum.model.resources.http.response
+
+import com.testerum.common.expression_evaluator.ExpressionEvaluator
+import com.testerum.model.resources.http.response.xml_body.XmlRootJsObject
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class XmlBodyHttpResponseTest {
```

Comment:
I like the tests. Good job.
Let's also add one that accesses an element that is not a valid JavaScript identifier (to force us to use the ``parent["element-name"]`` syntax).